src/legacy/core_plugins/kibana/public/home/kibana_services.ts

```diff
@@ -55,10 +55,14 @@ export function getServices() {
     indexPatternService: data.indexPatterns.indexPatterns,
     shouldShowTelemetryOptIn,
     telemetryOptInProvider,
-    getFeatureCatalogueRegistryProvider: async () => {
+    getFeatureCatalogueEntries: async () => {
       const injector = await chrome.dangerouslyGetActiveInjector();
       const Private = injector.get<IPrivate>('Private');
-      return Private(FeatureCatalogueRegistryProvider as any);
+      // Merge legacy registry with new registry
+      (Private(FeatureCatalogueRegistryProvider as any) as any).inTitleOrder.map(
+        npSetup.plugins.feature_catalogue.register
+      );
+      return npStart.plugins.feature_catalogue.get();
```

Comment:
@flash1293 Any issues with doing this here?

Reply:
Not all, that's fine 👍